expo-maps/android/src/main/java/expo/modules/maps/records/CameraPositionRecord.kt

```diff
@@ -0,0 +1,29 @@
+package expo.modules.maps.records
+
+import com.google.android.gms.maps.model.CameraPosition
+import com.google.android.gms.maps.model.LatLng
+import expo.modules.kotlin.records.Field
+import expo.modules.kotlin.records.Record
+
+class CameraPositionRecord(cameraPosition: CameraPosition) : Record {
+
+  @Field
+  var target: LatLngRecord = LatLngRecord(LatLng(0.0, 0.0))
+
+  @Field
+  var zoom: Float = 0.0f
+
+  @Field
+  var bearing: Float = 0.0f
+
+  @Field
+  var tilt: Float = 0.0f
+
+  init {
+    target = LatLngRecord(cameraPosition.target)
+    zoom = cameraPosition.zoom
+    bearing = cameraPosition.bearing
+    tilt = cameraPosition.tilt
+  }
```

Comment:
```suggestion
  @Field
  var target: LatLngRecord = LatLngRecord(cameraPosition.target)

  @Field
  var zoom: Float = cameraPosition.zoom

  @Field
  var bearing: Float = cameraPosition.bearing

  @Field
  var tilt: Float = cameraPosition.tilt
```
I think that code can be simplified like this^^^

Reply:
Yup, that looks way better!